### ISSUE TITLE
Authentication failed

### ISSUE THREAD

**Issue by alexisleblanc** [welaika/wordmove#153]
Hey there.
First thanks for this awesome wordmove, I can't wait to make it work haha.

Everything seems to work fine except the database. If I upload images for examples, it goes from my local to the server no problemo. I connect via SSH.

But for the Databse, I get this error:
▬▬ ✓ Pushing Database ▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬
   remote | mysqldump --host=localhost --user=user --password=password prodaqua_db > apps/prodaqua/public/wp-content/dump.sql
/Users/aleblanc/.rvm/gems/ruby-2.1.0/gems/net-ssh-2.9.1/lib/net/ssh.rb:219:in `start': Authentication failed for user serverpilot@104.236.255.77 (Net::SSH::AuthenticationFailed)

Any solution for me ? I don't understand because wordmove can connect without problem when I transfer other stuff but can't seem to connect for the database changes.

**Comment by alexisleblanc** [comment]
Ok I got it working.
I simply update my ruby version to 2.2.2
Then I got the mysqldump command not found.
I'm using DesktopServer so I added this to my Path and everything worked like a charm.
export PATH=$PATH:/Applications/XAMPP/xamppfiles/bin

This is my first post to Github, what do we do with issue that are solved ? Should we delete this one or leave it for future users ?

Thanks for the awesome Wordmove !
